travisCI/install-boost.sh

```diff
@@ -1,4 +1,30 @@
 #!/bin/bash
 set -exv
-wget 'http://downloads.sourceforge.net/project/boost/boost/1.58.0/boost_1_58_0.tar.gz'
-tar -xzf boost_1_58_0.tar.gz
+
+LIBS="accumulators algorithm align any array assert assign atomic bimap bind chrono concept_check config container conversion core date_time detail dynamic_bitset exception foreach function function_types functional fusion graph graph_parallel integer intrusive io iterator lexical_cast math move mpi mpl multi_array multi_index optional parameter predef preprocessor property_map proto python range ratio rational regex serialization smart_ptr spirit static_assert system test thread throw_exception timer tokenizer tti tuple type_index type_traits typeof unordered utility xpressive"
+TOOLS="build inspect"
+EXTRA="interval numeric_conversion ublas"
+
+# download required files
+wget https://github.com/boostorg/boost/archive/boost-1.61.0.tar.gz -O boost.tar.gz
```

Comment:
It would be nice to set the boost version via a variable. This would make switching to different versions easier.